skins/diadata.scss

```
@@ -0,0 +1,15 @@
+.diadata, .domo {
+    --header-bg: #141414;
+  }
+  .diadata {
+    --primary-color: #4f33ff;
+    --bg-color: #141414;
+    --text-color: #CCE0F4;
+    --link-color: #ff7096;
+    --heading-color: #CCE0F4;
+    --border-color: #00CCB7;
+  }
+  .dforce, .diadata {
+    --block-bg: transparent;
+  }
```

Please use one space per css file.
```suggestion
.diadata {
    --primary-color: #4f33ff;
    --bg-color: #141414;
    --text-color: #CCE0F4;
    --link-color: #ff7096;
    --heading-color: #CCE0F4;
    --border-color: #00CCB7;
  }
```

---

Done!